brave/common/extensions/url_bindings.cc

```diff
@@ -260,7 +260,11 @@ URLBindings::URLBindings(extensions::ScriptContext* context)
   RouteFunction("New",
             base::Bind(&URLBindings::New, base::Unretained(this)));
   RouteFunction("FormatForDisplay",
-            base::Bind(&URLBindings::FormatForDisplay, base::Unretained(this)));
+            base::Bind(
+              static_cast<void(URLBindings::*)(
```

Comment:
I think you probably had to add this cast because of the invalid method sig for the overloaded `FormatForDisplay` with `UnescapeRule`. See `FileBindings::WriteImportantFile` for an example with optional args and converters